index.rst

```diff
@@ -74,15 +119,16 @@ The manual can be found on Docushare `here <https://docushare.lsst.org/docushare
 
 Fiber Spectrograph
 ------------------
-The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits from the monochromator. 
+The fiber spectrograph used is an Avantes SensLine AvaSpecULS2048x64TEC-EVO, with a wavelength range of 200-1160 nm. An optical fiber runs from the fiber spectrograph and samples the light from the optical path after it exits the monochromator. 
 
-The fiber spectrograph is controlled via USB that runs directly from the electrometer to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
+The fiber spectrograph is controlled via USB that runs directly from the fiber spectrograph to a linux control computer (loonie) in the electronics cabinet. It can be commanded via CSC. 
```

Comment:
Please remove all references to Loonie and replace it with the new name (auxtel-ill-control.cp.lsst.org). 
Note that there is an instance of it in a figure as well.

Reply:
done